src/patternfly/upgrade-examples/LoginUpgradeExamples/docs/code.md

```diff
@@ -0,0 +1,12 @@
+## Overview
+
+When converting PatternFly 3 components to PatternFly 4 components, you must also take into consideration the layouts and sizings that PatternFly 3 utilized from Bootstrap 3.
+
+## Usage
+
+| PF3 Class | Applied To | PF4 Class | Applied To | Outcome |
+| -- | -- | -- | -- | -- |
+| `.login-pf` | `<div>` | `.pf-c-login` | `<div>` | Creates a login containing content. |
+| `.login-pf-heading` | `<div>` | `.pf-c-login__header` | `.pf-c-login` | Creates the header of a login. |    
+| `.login-pf-body` | `<div>` | `.pf-c-login__body` | `.pf-c-login` | Creates the body of a login. Required. | 
+| `.login-pf-footer` | `<div>` | `.pf-c-login__footer` | `.pf-c-login` | Creates the footer of a login. | 
```

Comment:
From the original issue, using https://rawgit.com/patternfly/patternfly/master-dist/dist/tests/login-basic.html as a reference, I'm not seeing these in pf3
* login-pf-body
* login-pf-heading
* login-pf-footer

I also see these in pf3, but not in the docs
* login-pf-page-header (corresponds to pf-l-login__header)
* login-pf-brand (pf-c-brand)
* login-pf-settings (pf-c-form__group.pf-m-action)
* login-pf-signup (no pf4 equivalent)
* login-pf-page-footer (pf-c-login__footer)
* login-pf-page-footer-links (pf-c-list.pf-m-inline)
* login-pf-page-footer-link (pf-c-login__footer-link)